x-pack/test/functional/services/observability/alerts/common.ts

```diff
@@ -25,6 +25,12 @@ const ACTION_COLUMN_INDEX = 1;
 
 type WorkflowStatus = 'open' | 'acknowledged' | 'closed';
 
+export enum AlertStatus {
+  all = 'all',
+  active = 'active',
```

Comment:
@ersin-erdal I suggest you import active and recovered values from https://github.com/elastic/kibana/blob/main/packages/kbn-rule-data-utils/src/alerts_as_data_status.ts

```
import {
  ALERT_STATUS_ACTIVE,
  ALERT_STATUS_RECOVERED,
} from '@kbn/rule-data-utils/alerts_as_data_status';
```

Reply:
i tried that but my IDE complained about it. it says that the folder is not in the root!

Reply:
Aaah, i import it directly! that's why it complained...
Fixing it. Thank you :)

Reply:
Done